test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -45,6 +45,19 @@ describe('Jetpack Compose', function () {
     await driver.settings().should.eventually.eql({ driver: 'compose' });
   });
 
+  it('should find element by xpath', async function () {
+    await driver.updateSettings({ driver: 'espresso' });
+    let el = await driver.elementByXPath("//*[@text='Clickable Component']");
+    await driver.moveTo(el);
+    await el.click();
+
+    await driver.updateSettings({ driver: 'compose' });
+
+    let e = await driver.elementByXPath("//*[@view-tag='lol']");
+    await e.text().should.eventually.equal('Click to see dialog');
+    await driver.back();
```

Comment:
driver.back should be called in a `finally` block to prevent following tests from failing